internal/v1/api.yaml

```diff
@@ -574,6 +574,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        users:
+          type: array
+          items:
+            $ref: '#/components/schemas/User'
+          description:
+            "list of users that a customer can add, also specifying their respective groups and SSH keys"
+    User:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: "user1"
+        SSHKey:
```

Comment:
Is there a usecase to create a user without an ssh key? I notice composer hasn't marked it as required either but wonder if there's any point to it.

Reply:
I think in IB we should require the SSH key (maybe composer too, but at least in IB it feels definitely right to be quite opinionated in what people can do).